setup.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -eq 0 ]
+  then echo "Run without sudo/root"
+  exit
+fi
+
+# Install Docker dependencies
+sudo apt-get -y install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
+sleep 1
```

Comment:
sleep here is unneccessary.

Reply:
Added --vm-drive params
Fixed minikube url to get the latest version
Removed sleeps
Added echo messages
Added kubectl context command

Reply:
Should the script delete any previous versions of minikube before starting the new one?

Something like:
```
minikube delete
sudo rm -rf ~/.kube
sudo rm -rf ~/.minikube
sudo rm -rf /etc/kubernetes
```